ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -64,6 +65,12 @@
       + "will not start serving requests until all preconditions are satisfied. Until that time, "
       + "requests will return a 503 error";
 
+  static final String KSQL_SERVER_ERRORS =
+          KSQL_CONFIG_PREFIX + "server.errors";
```

Comment:
Given the change in interface name, maybe:

```suggestion
          KSQL_CONFIG_PREFIX + "server.error.messages";
```